seed/utils/sensors.py

```diff
@@ -51,8 +51,7 @@ def _usages_by_exact_times(self, page, per_page):
         if self.showOnlyOccupiedReadings:
             sensor_readings = sensor_readings.filter(is_occupied=True)
 
-        # order by id **greatly** speeds this up (cause of indexing, I think
```

Comment:
you had me make a commit _just_ to add this comment, why are you removing it?

Reply:
Because that change is gone now in favor of a better one - also, you merged your PR before it was reviewed/approved, so I couldn't make the change there.